src/all/comickfun/src/eu/kanade/tachiyomi/extension/all/comickfun/Comick.kt

```diff
@@ -301,7 +301,7 @@ abstract class Comick(
                     is TagFilter -> {
                         if (it.state.isNotEmpty()) {
                             it.state.split(",").forEach {
-                                addQueryParameter("tags", it.trim())
+                                addQueryParameter("tags", it.trim().lowercase().replace(Regex("[ /]"), "-").replace("'-", "-and-039-").replace("'", "-and-039-"))
```

Comment:
take out the regex object, so it doesn't get created every time

Reply:
Hello, sorry for the late reply! Thank you for the suggestion, I've changed it the way you suggested. Is it correct? Sorry if I'm wrong

Reply:
Screenshot before and after build:
![ss](https://github.com/keiyoushi/extensions-source/assets/65448230/8ad971a7-4ed8-48ec-ab8e-933616fc80bc)